spec/copy-plugin-mock.js

```diff
@@ -0,0 +1,25 @@
+function FakeCopyWebpackPlugin() {};
+
+/**
+ * Mock plugins that modifies `compilation.assets`
+ *
+ * @param compiler
+ */
+FakeCopyWebpackPlugin.prototype.apply = function(compiler) {
+    compiler.plugin('emit', function(compilation, callback) {
+
+        var compiledMock = '// some compilation result\n';
+        compilation.assets['third.party.js'] = {
+            size: function() {
+                return compiledMock.length;
+            },
+            source: function() {
+                return compiledMock;
+            }
+        };
+
+        callback();
+    });
+};
+
+module.exports = FakeCopyWebpackPlugin;
```

Comment:
thanks for the suggestion! I have added a new tool to my belt :) ... Applied the formatting to relevant files.